pilot/pkg/networking/core/v1alpha3/listener_test.go

```diff
@@ -40,47 +42,66 @@ func TestOutboundListenerConflict_HTTP(t *testing.T) {
 	// The oldest service port is TCP.  We should encounter conflicts when attempting to add the HTTP ports. Purposely
 	// storing the services out of time order to test that it's being sorted properly.
 	testOutboundListenerConflict(t,
-		buildService(model.ProtocolHTTP, tnow.Add(1*time.Second)),
-		buildService(model.ProtocolTCP, tnow),
-		buildService(model.ProtocolHTTP, tnow.Add(2*time.Second)))
+		buildService("test1,com", model.ProtocolHTTP, tnow.Add(1*time.Second)),
+		buildService("test2,com", model.ProtocolTCP, tnow),
+		buildService("test3,com", model.ProtocolHTTP, tnow.Add(2*time.Second)))
 }
 
 func TestOutboundListenerConflict_TCP(t *testing.T) {
 	// The oldest service port is HTTP.  We should encounter conflicts when attempting to add the TCP ports. Purposely
 	// storing the services out of time order to test that it's being sorted properly.
 	testOutboundListenerConflict(t,
-		buildService(model.ProtocolTCP, tnow.Add(1*time.Second)),
-		buildService(model.ProtocolHTTP, tnow),
-		buildService(model.ProtocolTCP, tnow.Add(2*time.Second)))
+		buildService("test1.com", model.ProtocolTCP, tnow.Add(1*time.Second)),
```

Comment:
Add IP and resolution as options here.

Reply:
it's not needed ATM ... could easily be expanded in the future when needed.